README.rst

```diff
@@ -184,7 +184,7 @@ You can find a quick demo of coala here:
 
 |asciicast|
 
-.. |asciicast| image:: https://asciinema.org/a/42968.png
+.. |asciicast| image:: https://raw.githubusercontent.com/coala/artwork/master/artwork/asciinema_demo/asciinema_demo.png
```

Comment:
I suspect that this is because raw.githubusercontent.com doesnt provide content types.
https://rawgit.com/ is a solution for that problem.